### ISSUE TITLE
How can I setup Server on AWS?

### ISSUE THREAD

**Issue by sunilgulabani** [mrniko/netty-socketio#512]
I tried to deploy on AWS EC2. Application shows it started but when I try to connect, it fails stating 404 not found. Security groups are enabled for port 9092. `curl` command works fine on EC2 terminal. Problem is with accessing from browser

**Comment by pablojr** [comment]
@sunilgulabani isn't this a question for AWS support team? If connecting from EC2 terminal ("internally"), it looks like some firewall restriction is active, or port forwarding rule is missing. Could you check that with AWS?
And if you receive a "404 not found" error in your browser it seems that a server other than netty-socketio is responding (although I'm not quite sure though)

**Comment by sunilgulabani** [comment]
@pablojr It was not issue of AWS. That's my bad that I stated `404` status code in issue. Actually it wasn't invoking the netty-socketio server. I captured `tcpdump` to check if request is reaching to EC2 server. It was reaching the EC2 server but as application was listening only on "127.0.0.1:9092" it fails to route request to our chat server application. Following changes I did to make it work:

When we create SocketIO Server with hostname as "localhost", application is only accessible from local environment. I changed the hostname parameter value to "0.0.0.0" and it worked seamlessly. Rest AWS security group need to be expose for port which we used.
